velox/type/DecimalUtil.h

```diff
@@ -147,6 +148,50 @@ class DecimalUtil {
     return static_cast<TOutput>(rescaledValue);
   }
 
+  /**
+   * Rescale a double value to decimal value.
+   *
+   * Use `folly::convertTo` to convert a double value to int128_t or int64_t, it
```

Comment:
Please use /// for comments in header file.
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#comments